pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -188,6 +190,13 @@ def load_data(self):
             )
         )
 
+    def load_uploads(self):
+        """
+        Copies upload fixtures into MEDIA_ROOT
+        """
+        print 'Copying fixtures from uploads directory to MEDIA_ROOT'
+        sh('cp -rf common/test/uploads/* test_root/uploads/')
```

Comment:
Should: Rather than doing this via shell, you should do it via python. 
Question: I also have a broader question on why this should be necessary. 
Nit: I think copying the whole directory is the kind of thing that becomes more expensive as the codebase grows. Would it make sense to have a stricter scope (e.g., only the file you care about?)
Should: I notice that you're not actually leveraging the environment `MEDIA_ROOT` setting being used by bok-choy here, but instead it is hardcoded. Should you reference the var? (I realize this may not be possible since it's a django setting, but I wanted to ask if you have any more-robust recommendations.) Again this makes me wonder why you'd want to copy assets anyway, rather than storing them in both places <gasp!>